utils/webpack.config.babel.js

```diff
@@ -1,11 +1,13 @@
-var fs = require("fs");
-var handlebars = require("handlebars");
-var path = require("path");
-var postcss = require("postcss");
-var webpack = require("webpack");
+let fs = require("fs");
```

Comment:
I think I'm going to have one diff in the future that switches all `require`s to use const.